app/src/main/java/com/impulse/afterdarrk/Main.java

```diff
@@ -25,6 +29,8 @@
 public class Main extends AppCompatActivity {
     public static int width, height; // Width and height of screen
 
+    public static Context context; // Context of the activity
```

Comment:
This was already done before you (see lines above) but I'm not a fan of commenting on the same line as the code. Creates longer lines, Would prefer it to be above

Reply:
bit nitpicky I know